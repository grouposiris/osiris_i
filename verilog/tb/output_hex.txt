00000093
00100113
00202023
002020A3
00202223
00201023
002010A3
00201223
00200023
002000A3
00200223
# I-Type Instructions
00000093
00100113
FFF00193
00F2F213
00A3F313
0074F413
0035E513
0096E613
0057E713
00214093
00824193
00F34293
00A42393
01452493
01E62593
00573693
00F0B793
0191B113
00111093
00221193
00331293
00145393
00255493
00365593
40175693
4020D793
4031D113
# R-Type Instructions
003100B3
00628233
009403B3
40C58533
40F706B3
403100B3
005271B3
0083F333
00B574B3
00E6E633
0020E7B3
005261B3
0083C333
00B544B3
00E6C633
# S-Type Instructions
00202023
00202223
00202423
00309023
00309123
00309323
00410023
004100A3
004101A3
# B-Type Instructions
00208463
FE418EE3
00628663
00839863
FEA49CE3
00C59A63
00E6CC63
FE17CAE3
00314E63
02525063
FE7358E3
02945263
02B56463
FED666E3
02F76663
0220F863
FE41F4E3
0262FA63
# U-Type Instructions
000120B7
00023137
000341B7
00045217
00056297
00067317
# J-Type Instructions
020000EF
FF1FF16F
040001EF
