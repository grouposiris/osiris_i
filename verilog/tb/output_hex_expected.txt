00000093
00100113
00202023
002020A3
00202223
00201023
002010A3
00201223
00200023
002000A3
00200223
# I-Type Instructions
00000093
00100113
FFF00193
00F2F213
00A3F313
0074F413
0035E513
0096E613
0057E713
00214093
00824193
00F34293
00A42393
01452493
01E62593
00575613
00F0D713
0191E113
00109013
0020A013
0030B013
0010C093
0020D113
0030E193
4010F213
4020F393
4030E113
# R-Type Instructions
003100B3
006282B3
00940333
40C584B3
40F70533
403100B3
005211B3
0083A233
00B4D2B3
00E5E333
0020F3B3
005211B3
008342B3
00B45333
00E564B3
# S-Type Instructions
00202023
00202223
00202423
00301023
00301123
00301323
00402023
004020A3
004021A3
# B-Type Instructions
00208663
FE418EE3
00609663
0080C663
FEA4CEE3
0140D663
0182E663
FE61EEE3
01C10263
020112E3
FE9326E3
024122E3
02813863
FEB54AE3
02C15863
03016C63
FEE17EE3
03418C63
# U-Type Instructions
12345037
234560B7
34567137
45678217
56789297
67890317
# J-Type Instructions
020000EF
FF0FF0EF
0400016F